src/server/logger.js

```diff
@@ -1,27 +1,97 @@
 'use strict';
 
-var levels = ['trace', 'info', 'log', 'debug', 'warn', 'error'],
-    debug = require('debug');
+const chalk = require('chalk');
+
+const LEVELS = ['trace', 'info', 'log', 'debug', 'warn', 'error'],
+    LEVELCOLORS = {
+        'trace': {
+            bg: 'bgBlack',
+            fg: 'white'
```

Comment:
white foreground assumes a black background. I think if we don't dictate the output log color the it would default to the user's terminal preferences which will make it look better (some terminals and desktop environments default to white bg)

Reply:
Default is now used instead